packages/http-client-csharp/generator/Microsoft.Generator.CSharp.sln

```diff
@@ -23,7 +23,11 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestProjects.Local.Tests", "TestProjects\Local\TestProjects.Local.Tests.csproj", "{BFE39EDA-ADA1-4F60-A47C-ADE88C67B3B7}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Generator.CSharp.ClientModel.Tests", "Microsoft.Generator.CSharp.ClientModel\test\Microsoft.Generator.CSharp.ClientModel.Tests.csproj", "{D30EC838-B0A0-44CD-A361-239FF256A176}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Generator.CSharp.ClientModel.Tests", "Microsoft.Generator.CSharp.ClientModel\test\Microsoft.Generator.CSharp.ClientModel.Tests.csproj", "{D30EC838-B0A0-44CD-A361-239FF256A176}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Generator.CSharp.Shared", "Microsoft.Generator.CSharp.Shared\src\Microsoft.Generator.CSharp.Shared.csproj", "{BCA7EC85-C04B-4AD8-94EE-3559C7F5C6CD}"
```

Comment:
lets not add shared source as a project, just simply have the source in a folder named Shared under Microsoft.Generator.CSharp/src.  Then in each project wants wants to share you add something like this.

```
<Compile Include="[relative shared folder path]/**/*.cs" LinkBase="Shared" />
```

Reply:
I see. Thank you for explaining this.